session/session.go

```diff
@@ -65,6 +67,8 @@ var (
 	defaultSessionIdleExpiredTimeout = NoExpirationTimeout
 	defaultTimeFormat                = "Mon Jan 2 15:04:05 -0700 MST 2006"
 	MaxUserSessions                  = 1
+	healthcheckInterval              = time.Second * 60
```

Comment:
do you think we could maybe make it configurable? technically it will mean that we are sending data from the user device, maybe lets give the user ability to at least enable or disable keep alive and even change the interval?

Reply:
we could, but at this stage, a hard-coded 60 seconds looks "good enough". The clients (UI, mender-cli) will use the value mender-connect sends, thus we can easily make it configurable in the future if we need to.